app/models/job_run.rb

```diff
@@ -59,9 +59,9 @@ def in_progress?
     (waiting? || running?)
   end
 
-  # the states that indicate a job is completed
-  def finished?
-    !in_progress?
+  # indicates if the discovery report job is ready for display and is avaiable (some jobs may fail, leaving no report)
+  def report_ready?
+    !in_progress? && output_location && File.exist?(output_location)
   end
 
```

Comment:
we don't really need a `finished?` method, but a method indicating a report is ready to download is helpful to use in the view...